src/strategies/DSL/conditions/nonceCondition.ts

```diff
@@ -1,11 +1,36 @@
 import { PrimitiveArgs } from '@brinkninja/types'
-import { NonceConditionArgs } from '@brink-sdk'
+import { NonceConditionArgs, nonceToBit } from '@brink-sdk'
 
-function nonceCondition (def: NonceConditionArgs): PrimitiveArgs[] {
+const STATE_USED = 'USED'
+const STATE_NOT_USED = 'NOT_USED'
 
-  // TODO: implement
+function nonceCondition ({state, nonce} : NonceConditionArgs): PrimitiveArgs[] {
+  const bit = nonceToBit({nonce})
 
-  return []
+  switch (state) {
+    case STATE_USED:
+      return [
+        {
+          functionName: 'requireBitUsed',
+          params: {
+            bitmapIndex: bit.index,
+            bit: bit.value,
+          }
+        }
+      ]
+    case STATE_NOT_USED:
+      return [
+        {
+          functionName: 'requireBitNotUsed',
+          params: {
+            bitmapIndex: bit.index,
+            bit: bit.value,
```

Comment:
@mikec do you mind checking what values we need for other parameters? 

* chainId
* signer

chainId is optional and signer is mandatory.